src/service/CompanyService.ts

```diff
@@ -176,6 +178,16 @@ export class CompanyService extends APIClient {
                 headers: { 'Content-Type': 'application/json' },
             });
     }
+
+    /**
+     * This ressource is used for setting the ownerClass of a company.
+     * @param {string} companyId - The companyId of the company that should be updated
+     * @param {string} ownerClass - The OwnerClass that the company should get. Values: "INTERNAL/CUSTOMER/EXTERNAL_DEVELOPER"
```

Comment:
```suggestion
     * @param {OwnerClass} ownerClass - The OwnerClass that the company should get. Values: "INTERNAL/CUSTOMER/EXTERNAL_DEVELOPER"
```

Kannst hier den oben definierten Typ nutzen